tools/python/liveanalytics_migration_scripts/targets/timestream_for_influxdb/transform/transform.py

```diff
@@ -643,6 +654,14 @@ def parse_table_dimensions(arg):
         required=True,
         type=parse_bool_cli_argument,
     )
+    parser.add_argument(
```

Comment:
Rather than using a flag, why not dynamically determine if the `time_ns` column exists and make it the default. I don't see any reason why we would want to use the `time` column if the `time_ns` column already exists.